its/src/test/java/com/sonar/it/scanner/msbuild/others/RuntimeConfigTest.java

```diff
@@ -34,21 +34,15 @@
 class RuntimeConfigTest {
 
   @Test
-  void TestRuntimeConfigRollForward() throws IOException {
-    Map<String, String> releasedSuffixAndTfm  = new HashMap<>() {{
-      put("netcoreapp3.0", "netcoreapp3.1");
-    }};
-
+  void TestRuntimeConfig() throws IOException {
     Gson gson = new Gson();
-    for (Map.Entry<String, String> suffixAndTfm : releasedSuffixAndTfm.entrySet()) {
-      Path runtimeConfigPath = Paths.get("..", "build", "sonarscanner-msbuild-" + suffixAndTfm.getKey(), "SonarScanner.MSBuild.runtimeconfig.json");
-      assertThat(runtimeConfigPath.toAbsolutePath()).exists();
-
-      Config config = gson.fromJson(Files.newBufferedReader(runtimeConfigPath), Config.class);
-      assertThat(config.runtimeOptions).isNotNull();
-      assertThat(config.runtimeOptions.tfm).isEqualToIgnoringCase(suffixAndTfm.getValue());
-      assertThat(config.runtimeOptions.rollForward).isEqualToIgnoringCase("LatestMajor");
-    }
+    Path runtimeConfigPath = Paths.get("..", "build", "sonarscanner-net", "SonarScanner.MSBuild.runtimeconfig.json");
+    assertThat(runtimeConfigPath.toAbsolutePath()).exists();
+
+    Config config = gson.fromJson(Files.newBufferedReader(runtimeConfigPath), Config.class);
```

Comment:
I am not sure `gson.fromJson` closes the `BufferedReader`.
Half of the do what it seems to be [the right thing](https://sourcegraph.com/github.com/dbeaver/dbeaver@518ed1a5b1c2d0806f38f3c11bd779331f09257f/-/blob/bundles/org.jkiss.utils/src/org/jkiss/utils/rest/RestServer.java?L214:27), whereas the other half don't use a `try`...